### PR TITLE
 Add test for `$val: null` not working.

### PR DESCRIPTION
**Pull request by bqrkhn**

### CODE REVIEW

File: src/pg/test/db/dbWrite.test.js [hunk 1]
```diff
@@ -191,4 +208,22 @@ describe('postgres', () => {
...
+    await store.write('sfWrite.foo', {
+      write: { WhatId: { $val: null } },
+    });
+
+    const sqlQuery = sql`
+      UPDATE "sfWrite" SET
+        "write" = case jsonb_typeof("write")
+          when 'object' then "write"
+          else '{}'::jsonb
+        end || jsonb_build_object(${'WhatId'}::text, ${'{"WhatId":null}'}::jsonb),
```
Comment (aravindet) [inline src/pg/test/db/dbWrite.test.js]: This is not correct, the expectation is:


```suggestion
        end || jsonb_build_object(${'WhatId'}::text, ${'{"$val":null}'}::jsonb),
```
